homework/12_A/10_Zlatina_Lilova/homework01/circles.py

```diff
@@ -0,0 +1,34 @@
+import math
+
+# the function returns the number of mutual points of the circles
+
+
+def circle_check(radius1, radius2, center_1, center_2):
+    a = center_1[1] - center_2[1]
+    b = center_1[0] - center_2[0]
+
+    c = math.sqrt(a*a + b*b)
+
+    if(radius2 > radius1):
+        rad = radius2 - radius1
+    else:
+        rad = radius1 - radius2
+
+    position = -1
+
+    if(c == 0 and radius2 == radius1):
+        print("The circles are matching")
```

Comment:
The prints should either be removed (if you added those for debug) or turned into a debug log.